src/components/views/UserHand.js

```diff
@@ -0,0 +1,139 @@
+                        }
+                    });
+                setBlackCard(response.data);
+            } catch (error) {
+                if (error.response.status === 401 || error.response.status === 404) {
+                    setBlackCard(null);
+                    console.error("Error ", error.response.status, ": ", error.response.message);
+                } else {
+                    console.error("Details: ", error);
+                    alert("Invalid Input:\n " + handleError(error));
+                }
+            }
+        }
+        getBlackCard();
+    }, []);
+
+    useEffect( () => {
+        async function getWhiteCards() {
+            try {
+                const response = await api.get(`/users/${userId}/games/whiteCards`,
+                    {
+                        headers: {
+                            "authorization": token
+                        }
+                    });
+                setWhiteCards(response.data);
+            } catch (error) {
+                if(error.response.status === 401 || error.response.status === 404) {
+                    setWhiteCards(null);
+                    console.error("Error " , error.response.status, ": ", error.response.message);
+                } else {
+                    console.error("Details: ", error);
+                    alert("Invalid Input:\n " + handleError(error));
+                }
+            }
+        }
+        getWhiteCards();
+    }, []);
+
+    let blackCardContent
+    let whiteCardsContent = <div className="hand">No more white Cards available</div>
+    let cardsOnPile;
+
+    if(blackCard) {
+        blackCardContent = <CardButton className="card blackCard" disabled={true}>
+            {blackCard.text}
+        </CardButton>
+    } else {
+        blackCardContent = <CardButton className="card blackCard"
+                                       onClick={() => {history.push('/game/select/blackCard')}}>
+            No black Card set, Choose black Card
+        </CardButton>
+    }
+
+    if (whiteCards){
+        if (whiteCards.length === 0 && !blackCard) {
+            whiteCardsContent = <CardButton className="card whiteCard simple"
+                                            onClick={() => {history.push('/game/select/blackCard')}}>
+                No white Card available, Did you choose a black Card?
+            </CardButton>
+        } else if (whiteCards.length === 0 && blackCard) {
+            whiteCardsContent = <CardButton className="card whiteCard" disabled={true}>
+                No more white Cards available for today
+            </CardButton>
+        } else if (whiteCards.length > 0) {
+            if (whiteCards.length > cardsPerHand) {
+                const diff = whiteCards.length - cardsPerHand;
+                cardsOnPile = <CardButton className="card whiteCard" disabled={true} style={{justifySelf: "flex-end"}}>
+                    You can draw {diff} more cards.
```

Comment:
Just something small:
Change that to ``You can draw {diff} more card{(diff === 1) ? "" : "s"``
(also in PlayWhites.js)